src/DetailsView/details-view-body.tsx

```diff
@@ -65,10 +67,19 @@ export interface DetailsViewBodyProps {
 
 export class DetailsViewBody extends React.Component<DetailsViewBodyProps> {
     public render(): JSX.Element {
+        const bodyLayoutClassname = classNames({
+            'details-view-body-nav-content-layout reflow-ui': this.props.featureFlagStoreData[
```

Comment:
You can simplify this

```
'details-view-body-nav-content-layout': true,

'reflow-ui': this.props.featureFlagStoreData[FeatureFlags.reflowUI]
```

Reply:
gotcha. added that in the latest commit.